__tests-pacts__/index.ts

```diff
@@ -546,6 +546,14 @@ const pactSpec: PactSpec = {
       ],
     ],
   },
+  ThreadEditCommentMutation: {
+    examples: [
+      [
+        { content: comment.content, commentId: comment.id, userId: user.id },
+        { success: true },
```

Comment:
What about removing it as you proposed? It's indeed a good idea, since it removes unnecessary code.
```suggestion
        undefined,
```